manifest/oci.go

```diff
@@ -225,6 +225,44 @@ func (m *OCI1) Inspect(configGetter func(types.BlobInfo) ([]byte, error)) (*type
 	return i, nil
 }
 
+// Inspect returns various information for (skopeo inspect) parsed from the manifest and configuration.
+func (m *OCI1) InspectV2(configGetter func(types.BlobInfo) ([]byte, error)) (*types.ImageInspectInfoV2, error) {
```

Comment:
I’m sorry about the lack of detail - it _is_ API-compatible to add new fields to the returned struct; we just can’t modify any existing ones.

So an entire new `InspectV2` isn’t necessary — and in fact the copy&pasted duplication would be harder to maintain.